### DISCUSSION TITLE
The mapping functions are handled in the same function？

### DISCUSSION THREAD

**Discussion by FTWH** [lithops-cloud/lithops#761] [Q&A] [answered]
Lithops with AWS Lambda backend handles user map function requests by using multiprocess lib to enable parallel processing.
However, according to AWS Docs, the lambda function memory size under 1769MB will not have the second CPU core. As a result, the mapping functions are actually not accelerated.

**Comment by JosepSampe** [comment, marked as answer]
Independently of the memory size, AWS Lambda provides the parallelism by starting a separate container for each activation of a `map()`. By default, each started container processes 1 element of the `iterable`. For example, if the `iterable` of a `map(func, itrerable)` contains 1000 elements, Lithops + AWS lambda will start 1000 separate containers, each one processing one element of the `iterable`.

For memory sizes > 1769MB, where you get more than one cpu per container, you can use the `worker_processes` config key to send multiple activation to the same worker, this way you will leverage all the CPUs in a container.
